Cargo.toml

```diff
@@ -47,8 +49,182 @@ exclude = [
     "mm2src/mm2_test_helpers",
 ]
 
-# https://doc.rust-lang.org/beta/cargo/reference/features.html#feature-resolver-version-2
-resolver = "2"
+[workspace.dependencies]
+aes = "0.8.3"
+argon2 = { version = "0.5.2", features = ["zeroize"] }
+arrayref = "0.3"
+anyhow = "1.0"
+async-std = "1.5"
+async-trait = "0.1"
+async-stream = "0.3"
+backtrace = "0.3"
+base64 = "0.21.2"
+bech32 = "0.9.1"
+bs58 = "0.4.0"
+bigdecimal = { version = "0.3", features = ["serde"] }
+bip32 = { version = "0.2.2", default-features = false, features = ["alloc", "secp256k1-ffi"] }
+bip39 = { version = "2.0.0", features = ["rand_core", "zeroize"], default-features = false }
+bitcoin = "0.29"
+bitcoin_hashes = "0.11"
+blake2 = "0.10.6"
+blake2b_simd = "0.5.10"
+bytes = "1.1"
+byteorder = "1.3"
+cbc = "0.1.2"
+cc = "1.0"
+cipher = "0.4.4"
+chrono = "0.4.23"
+cfg-if = "1.0"
+clap = { version = "4.2", features = ["derive"] }
+cosmrs = { version = "0.16", default-features = false }
+crossbeam = "0.8"
+crossbeam-channel = "0.5.1"
+compatible-time = { version = "1.1.0", package = "web-time" }
+crc32fast = { version = "1.3.2", features = ["std", "nightly"] }
+derive_more = "0.99"
+directories = "5.0"
+dirs = "1"
+ed25519-dalek = { version = "1.0.1", features = ["serde"] }
+either = "1.6"
+enum-primitive-derive = "0.2"
+env_logger = "0.9.3"
+ethabi = "17.0.0"
+ethcore-transaction = { git = "https://github.com/KomodoPlatform/mm2-parity-ethereum.git", rev = "mm2-v2.1.1" }
+ethereum-types = { version = "0.13", default-features = false, features = ["std", "serialize"] }
+ethkey = { git = "https://github.com/KomodoPlatform/mm2-parity-ethereum.git", rev = "mm2-v2.1.1" }
+# Waiting for https://github.com/rust-lang/rust/issues/54725 to use on Stable.
+#enum_dispatch = "0.1"
+ff = "0.8"
+findshlibs = "0.5"
+# using select macro requires the crate to be named futures, compilation failed with futures03 name
+futures = { version = "0.3.1", default-features = false }
+futures01 = { version = "0.1", package = "futures" }
+futures-rustls = { version = "0.24", default-features = false }
+futures-ticker = "0.0.3"
+futures-timer = "3.0"
+futures-util = { version = "0.3", default-features = false, features = ["sink", "std"] }
+fnv = "1.0.6"
+group = "0.8.0"
+gstuff = { version = "0.7", features = ["nightly"] }
+hash256-std-hasher = "0.15.2"
+hash-db = "0.15.2"
+hex = "0.4.2"
+hmac = "0.12.1"
+http = "0.2"
+http-body = "0.4"
+httparse = "1.8.0"
+hyper = "0.14.26"
+# using webpki-tokio to avoid rejecting valid certificates
+# got "invalid certificate: UnknownIssuer" for https://ropsten.infura.io on iOS using default-features
```

Comment:
this doesn't fit here as the feature `webpki-tokio` is mentioned in children `.toml`s